raspberry-pi/4/poe-hat.nix

```diff
@@ -16,7 +16,7 @@ in {
 
     hardware.deviceTree = {
       overlays = [
-        # Equivalent to: https://github.com/raspberrypi/linux/blob/rpi-5.15.y/arch/arm/boot/dts/overlays/rpi-poe-overlay.dts
+        # Equivalent to: https://github.com/raspberrypi/linux/blob/rpi-6.1.y/arch/arm/boot/dts/overlays/rpi-poe-overlay.dts
```

Comment:
Can you set an assertion like here: https://github.com/NixOS/nixpkgs/blob/8a05bed379935ad7e50b338edd09f7f13c996c6a/nixos/modules/services/logging/journalbeat.nix#L67
to make sure the linux is >= 6.1 (i.e. `(lib.versionAtLeast config.boot.kernelPackages.kernel "6.1")`)

Reply:
Than people on stable can upgrade.